src/patternfly/components/Menu/menu-label.hbs

```diff
@@ -0,0 +1,17 @@
+<label class="pf-c-menu__label{{#if menu-label--modifier}} {{menu-label--modifier}}{{/if}}"
```

Comment:
I wonder if we could make the `__item` element the `label`, which would be more inline with how the other menu items work where the `__item` is the clickable thing. Is there a reason that wouldn't work? Something like this:
<img width="589" alt="Screen Shot 2022-02-25 at 1 26 58 PM" src="https://user-images.githubusercontent.com/35148959/155776363-377bfb0c-1abe-4d18-b071-8ba249a21bd8.png">

Reply:
Aside: have to add `role="menuitem"` to `.pf-c-menu__list-item` containing a checkbox.